docs/source/GetStartedGuide.rst

```diff
@@ -216,6 +216,23 @@ instrumented and simple tests::
     JOB HTML  : $HOME/avocado/job-results/job-2014-08-12T15.42-86911e49/html/results.html
     TESTS TIME : 5.88 s
 
+Interrupting The Job On First Failed Test (failfast)
+====================================================
+
+The Avocado ``run`` command has the option ``--failfast`` to exit the job on
+first failed test::
+
+    $ avocado run --failfast /bin/true /bin/false /bin/true /bin/true
+    JOB ID     : eaf51b8c7d6be966bdf5562c9611b1ec2db3f68a
+    JOB LOG    : $HOME/avocado/job-results/job-2016-07-19T09.43-eaf51b8/job.log
+    TESTS      : 4
+     (1/4) /bin/true: PASS (0.01 s)
+     (2/4) /bin/false: FAIL (0.01 s)
+    Interrupting job (failfast).
```

Comment:
That `INTERRUPTED (0.92 s)` you're mentioning is the test status, printed after a line break. Job 'says' nothing on `INTERRUPTED` test. Our test status here is already printed (`FAIL (0.01 s)`). Failfast is a different case, were the Job has to 'speak' so user can know what happened. I like the current message... I can put it in red, but it would mean the message is an error, which is not true. Thoughts?